tools/deploy-with-olm.sh

```diff
@@ -41,17 +41,18 @@ TARGET_NAMESPACE=${TARGET_NAMESPACE:-openshift-storage}
 
 # CSV environment variables
 ADDON_NAME=${ADDON_NAME:-ocs-converged}
-BUNDLE_FILE=bundle/manifests/ocs-osd-deployer.clusterserviceversion.yaml
+BUNDLE_FILE=${OUTPUT_DIR}/manifests/ocs-osd-deployer.clusterserviceversion.yaml
 CLUSTER_SIZE=${CLUSTER_SIZE:-1}
 
 # Generate the deployer image 
 blue "Generate deployer image: ${DEPLOYER_IMAGE}"
 exit_on_err make docker-build IMG=${DEPLOYER_IMAGE}
 exit_on_err make docker-push IMG=${DEPLOYER_IMAGE}
 
+OUTPUT_DIR=./tools/bundle
```

Comment:
You are defining this value after you already use it (line 44). Please move this immediately before line 44